cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -326,4 +327,16 @@ export class Assessment {
         cy.get(commonView.sideDrawer.labelContent).contains(risk);
         click(commonView.sideDrawer.closeDrawer);
     }
+    public static discardAssessments(): void {
```

Comment:
Could you rename this to discardAssessmentsAssessmentActions() and the corresponding archetype and application method to be more specific and to distinguish this from the other way of discarding assessment through the kebab menu ?

Reply:
Sure done!